barretenberg/cpp/src/barretenberg/protogalaxy/decider_prover.cpp

```diff
@@ -17,7 +17,7 @@ DeciderProver_<Flavor>::DeciderProver_(const std::shared_ptr<Instance>& inst,
                                        const std::shared_ptr<Transcript>& transcript)
     : accumulator(std::move(inst))
     , transcript(transcript)
-    , commitment_key(inst->commitment_key)
+    , commitment_key(inst->proving_key->commitment_key)
```

Comment:
storing commitment_key is really only useful because now we don't have to write` instance->proving_key->commitment_key` everywhere we want to use the commitment_key, but this should probably be changed to a reference instead of a shared_ptr

Reply:
Makes sense, feel free resolve in a future PR.